app/src/main/java/com/example/mymovieinfo/MainActivity.kt

```diff
@@ -2,11 +2,34 @@ package com.example.mymovieinfo
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.Button
+import android.widget.TextView
+import android.widget.Toast
+
+import org.w3c.dom.Text
+import java.io.IOException
+import java.io.InputStream
+
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-    //test 2
+
+        var year = findViewById<TextView>(R.id.txt_Year)
+        var name = findViewById<TextView>(R.id.txt_MovieName)
+        var result = findViewById<TextView>(R.id.txt_Results)
+
+
+        val btnFindMyMovie = findViewById<Button>(R.id.btn_FindMyMovie)
+        btnFindMyMovie.setOnClickListener {
+            // your code to perform when the user clicks on the button
+
+            result.text = "the $name was released in $year"
+
+
+            Toast.makeText(this@MainActivity, "You clicked me.", Toast.LENGTH_SHORT).show()
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.